.circleci/config.yml

```diff
@@ -30,24 +31,107 @@ jobs:
+      - persist_to_workspace:
+          root: ~/project
+          paths:
+            - "vendor/bundle"
+  install_node_deps:
+    executor: my-executor
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+            - npm-cache-{{ checksum "package-lock.json" }}
       - run:
           name: Install node dependencies
           command: npm install
-      - run:
-          name: Build USWDS if needed
-          command: npm run build-uswds
       - save_cache:
           key: npm-cache-{{ checksum "package-lock.json" }}
           paths:
             - node_modules
+  build:
+    executor: my-executor
+    steps:
+      - checkout
+      - attach_workspace:
+          at: ~/project
+      - restore_cache:
+          keys:
+            - npm-cache-{{ checksum "package-lock.json" }}
+      - restore_cache:
+          keys:
+            - gem-cache-{{ checksum "Gemfile.lock" }}
+            - gem-cache
+      - run:
+          name: Build USWDS if needed
+          command: npm run build-uswds
+      - run:
+          name: Build site assets
+          command: npm run build-all-assets
+      - persist_to_workspace:
+          root: ~/project
+          paths:
+            - "_site"
+            - "assets"
+            - "node_modules"
+  test_build:
+    executor: my-executor
+    steps:
+      - checkout
+      - attach_workspace:
+          at: ~/project
+      - run:
+          name: Run rspec
+          command: |
+            bundle config set --local path '~/project/vendor/bundle'
+            bundle exec rspec
+      - run:
+          name: Run HTML proofer
+          command: npm run proof
       - snyk/scan:
           organization: uswds
+  test_a11y_desktop:
+    executor: my-executor
+    steps:
+      - checkout
+      - attach_workspace:
+          at: ~/project
+      - run:
+          name: Link bundler
+          command: |
+            bundle config set --local path '~/project/vendor/bundle'
+      - run:
+          name: Run pa11y-ci desktop
+          command: npm run start-detached && npm run pa11y-ci:sitemap
+  test_a11y_mobile:
+    executor: my-executor
+    steps:
+      - checkout
+      - attach_workspace:
+          at: ~/project
+      - run:
+          name: Link bundler
+          command: |
+            bundle config set --local path '~/project/vendor/bundle'
       - run:
-          name: Run test
-          command: npm run test:ci
+          name: Run pa11y-ci mobile
+          command: npm run start-detached && npm run pa11y-ci:sitemap-mobile
 
 workflows:
-  version: 2
   circle-uswds-site:
     jobs:
-      - build
+      - install_ruby_deps
+      - install_node_deps
+      - build:
+          requires:
+            - install_ruby_deps
+            - install_node_deps
+      - test_build:
+          requires:
+            - build
+      - test_a11y_desktop:
+          requires:
+            - build
+      - test_a11y_mobile:
+          requires:
+            - build
```

Comment:
Separated pa11y tests so they can run at the same time

![image](https://user-images.githubusercontent.com/3385219/167209622-b191d8b6-223b-459b-ae9c-f6f9ce75efc7.png)

Reply:
Hallelujah!